metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures datadog components'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '3.3.0'
+version          '3.99.0'
```

Comment:
I was thinking about 3.99.1 for fixes, and keep 3.4.0 available in case we want to actually release another 3.x version. I don't think this is important as long as we don't actually release a 3.99 to the public.

Reply:
Definitely made this review too early today. `3.99.1` will work. 🙇